.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "ion-object-mapper-dotnet"]
```

Comment:
This is temporary to get the build passing. Will be removed once Ion object mapper repos is on NuGet.